public_dropin_gpu_environments/nemollm_inference_ms/Dockerfile

```diff
@@ -71,10 +72,12 @@ RUN apt-get update && apt-get install -y \
   && rm -rf /var/lib/apt/lists/*
 
 USER 1000
+ENV DATAROBOT_VENV_PATH=/home/nemo/dr
 ENV PIP_NO_CACHE_DIR=1
-RUN python -m venv /home/nemo/dr && /home/nemo/dr/bin/pip install -U pip
+ENV PATH=${BASE_PATH}
```

Comment:
Why are you setting this again? You already set it on line 17?

Reply:
env vars were exported for `root` in the build, then you changed to the nemo user. is docker expected to set the same env vars for both users?

I had issues with the path, nemo user was unable to find `mpirun`.

Reply:
Oh hmm, I thought env vars were global to the image and not based on `USER` 😕